docs/source/ephys_at_swc/community.md

```diff
@@ -0,0 +1,21 @@
+# Community
+
+This site is maintained by the
+[NeuroInformatics Unit](https://neuroinformatics.dev/).
+Please don't hesitate to contact us
+(in particular Joe Ziminski) on Slack with any questions or feedback.
```

Comment:
Maybe link to the Zulip here so any external people can get in touch too?

Reply:
Nice yes